### PR TITLE
Fixed bug in 91-bug-in-widget-textupdatemultiplepvscomponent

### LINKED ISSUE #91: Bug in widget TextUpdateMultiplePVsComponent
TextUpdateMultiplePVsComponent

Is not displaying the correct output

### PR DESCRIPTION
**Pull request by wduckitt**
Closes #91

TextUpdateMultiplePVsComponent

was not displaying the correct output.

This pull request fix it

### CODE REVIEW

**Comment by hanak** [comment]
I am not sure I can spot the fix. Unfortunately, there is no description of neither the problem nor the fix. Would it be possible to give me some hints, please?

File: ReactApp/src/components/BaseComponents/TextUpdateMultiplePVs.js [hunk 1]
```diff
@@ -44,50 +42,53 @@ const TextUpdateMultiplePVsComponent=(props)=> {
...
  * The component is implemented with zero margins and enabled to grow to the width of its parent container.<br/><br/>
  * The margins and spacing must be controlled from the parent component.<br/><br/>
  */
-const TextUpdateMultiplePVs =(props)=>{
-  return (
-    <Widget {...props} component={TextUpdateMultiplePVsComponent} pvs={undefined}/>
-  )
-}
+const TextUpdateMultiplePVs = (props) => {
+  return <Widget {...props} component={TextUpdateMultiplePVsComponent} />;
```
Comment (wduckitt) [inline ReactApp/src/components/BaseComponents/TextUpdateMultiplePVs.js:86]: The change on this line was the bug fix

Comment (wduckitt) [reply ReactApp/src/components/BaseComponents/TextUpdateMultiplePVs.js:86]: It seems I broke it in a previous release, and never picked it up

Comment (wduckitt) [reply ReactApp/src/components/BaseComponents/TextUpdateMultiplePVs.js:86]: It's not a component I ever used so I never picked up the mistake

Comment (hanak) [reply ReactApp/src/components/BaseComponents/TextUpdateMultiplePVs.js:86]: Thank you.

**Review by hanak — APPROVED** [review]
Change looks good. Approved.

**Pull request merged**